src/components/PropEditors.js

```diff
@@ -400,9 +391,7 @@ export function makeLocalizedLocationField(intl, property) {
   let label = intl.formatMessage({ id: property });
```

Comment:
Getting the following error when trying to edit my profile
<img width="851" alt="screen shot 2017-09-18 at 6 41 33 pm" src="https://user-images.githubusercontent.com/4016496/30572255-58e1989c-9ca1-11e7-9960-4a2204558419.png">